.gitignore

```diff
@@ -11,3 +11,5 @@ coverage
 # logs
 .eslintcache
```

Comment:
this is a macOS thing. i still don't fully understand what triggers it being included in a commit, but adding to the .gitignore as it does not include any project-relevant or project-critical info